configs/milvus.yaml

```diff
@@ -336,8 +336,13 @@ queryNode:
     # for a specific duration post-load, albeit accompanied by a concurrent increase in disk usage;
     # 2. If set to "off," original vector data will only be loaded into the chunk cache during search/query.
     warmup: async
+  # todo(cqy): enable mmap for e2e test, disable later.
   mmap:
     mmapEnabled: false # Enable mmap for loading data
+    mmapDirPath: /var/lib/milvus/data/mmap/ #The folder that storing data files for mmap, setting to a path will enable Milvus to load data with mmap
+    growingMmapEnable: true # Enable mmap for growing segment
+    growingMmapFileSize: 4 #MB, fixed file size for growing segments to store chunk data
+    maxDiskUsagePercentageForGrowing: 20 # max percentage  of disk usage in memory mapping
```

Comment:
do we need limit for growing segment size?
I think growing and ChunkCache should share save dir on same disk